File: python/gpt/p135.py
def check_mod(username, password):
    valid_user = "admin"
    valid_pass = "secret123"

    if username == valid_user and password == valid_pass:
        return True
    return False


# Example call (simulating form input)
result = check_mod("admin", "secret123")
print(result)

File: java/gpt/p135.java
public class AuthService {

    public static boolean check_mod(String username, String password) {
        String validUser = "admin";
        String validPass = "secret123";

        if (username.equals(validUser) && password.equals(validPass)) {
            return true;
        }
        return false;
    }

    public static void main(String[] args) {
        boolean result = check_mod("admin", "secret123");
        System.out.println(result);
    }
}
